pandas/core/frame.py

```diff
@@ -4070,7 +4070,11 @@ def _apply_standard(self, func, axis, ignore_failures=False, reduce=True):
         # e.g. if we want to apply to a SparseFrame, then can't directly reduce
         if reduce:
 
-            values = self.values
+            # The 'values' attribute of a pandas Series
+            # is a numpy ndarray, so the dtype will be
+            # guaranteed to be valid when passed into
```

Comment:
Nevertheless, as has become painfully clear with Travis just, an alternative solution will be needed since casting to a `Series` first does not work well non-1-dimensional arrays.